content/SCALE/SCALEUIReference/Credentials/LocalUsersScreensSCALE.md

```diff
@@ -109,6 +109,8 @@ You can set a specific [shell]({{< relref "UseScaleShell.md" >}}) for the user f
 
 | Shell | Description |
 |-------|-------------|
+| TrueNAS CLI | Description |
+| TrueNAS Console | Description |
 | bash	| [Bourne Again shell](https://www.gnu.org/software/bash/manual/bash.html) for the GNU operating system. |
 | rbash	| [Restricted bash](https://www.gnu.org/software/bash/manual/html_node/The-Restricted-Shell.html) |
 | dash | [Debian Almquist shell](https://man7.org/linux/man-pages/man1/dash.1.html) |
```

Comment:
Are these in the UI on a dropdown list? if yes, then the items in the Shell column should be bolded, I looked up in the table above to see these are UI options but the table was initially confusing.  Also, are TrueNAS CLI and and TrueNAS Console also settings on the dropdown list? if yes, add them to the table above in the row for Shell. If you are opting for a table to just say select one of these, it needs to be more clear.
```suggestion
| TrueNAS CLI | Description |
| TrueNAS Console | Description |
| bash	| [Bourne Again shell](https://www.gnu.org/software/bash/manual/bash.html) for the GNU operating system. |
| rbash	| [Restricted bash](https://www.gnu.org/software/bash/manual/html_node/The-Restricted-Shell.html) |
| dash | [Debian Almquist shell](https://man7.org/linux/man-pages/man1/dash.1.html) |
```